services/packages/server/packages_test.go

```diff
@@ -646,6 +646,7 @@ func TestListInstalled(t *testing.T) {
 	testdataInputBad2 := "./testdata/yum-installed-bad2.out"
 	testdataInputBad3 := "./testdata/yum-installed-bad3.out"
 	testdataGolden := "./testdata/yum-installed.textproto"
+	testdataOldVersion := "./testdata/yum-installed-old-server-version.textproto"
```

Comment:
Textprotos aren't very effective for this sort of test (https://protobuf.dev/programming-guides/dos-donts/#dont-use-text-format-messages-for-interchange). If you want to do a compatibility test, the testdata should be the binary proto encoding so that we're testing the on-the-wire format.

A better long-term option is adding a separate lint for protobuf changes, something like https://github.com/nilslice/protolock. We can do that as a separate PR.